app/controllers/general.php

```diff
@@ -289,19 +291,74 @@
         ->noContent();
 }, ['request', 'response']);
 
-App::error(function ($error, $utopia, $request, $response, $layout, $project) {
+App::error(function ($error, $utopia, $request, $response, $layout, $project, $logger, $loggerBreadcrumbs) {
     /** @var Exception $error */
     /** @var Utopia\App $utopia */
     /** @var Utopia\Swoole\Request $request */
     /** @var Appwrite\Utopia\Response $response */
     /** @var Utopia\View $layout */
     /** @var Appwrite\Database\Document $project */
+    /** @var Utopia\Logger\Logger $logger */
+    /** @var Utopia\Logger\Log\Breadcrumb[] $loggerBreadcrumbs */
+
+    $user = null;
```

Comment:
No need to assign `null` here, If you use `isset()` later on to check if a user is set 👍🏻

Reply:
Applied, thanks.